openlibrary/plugins/upstream/borrow.py

```diff
@@ -189,6 +190,12 @@ def POST(self, key):
             except lending.PatronAccessException as e:
                 stats.increment('ol.loans.blocked')
 
+                add_flash_message(
+                    'error',
+                    _('Unable to complete borrow action. Please try again later.'),
```

Comment:
@mekarpeles you'll probably want to update this copy.  I think that you suggested something specific last week, but I was unable to find the suggestion.

Reply:
@bfalling your input required